src/Notes.h

```diff
@@ -247,7 +249,7 @@ class NotePairs: public RefCountable
 
     /// Convert NotePairs list to a string consist of "Key: Value"
     /// entries separated by sep string.
-    const char *toString(const char *sep = "\r\n") const;
+    SBuf toString(const char *sep = "\r\n") const;
```

Comment:
Just for my information, does PR code ever call this method without arguments (i.e. relying on the CRLF default)? I know that we provide a CRLF argument in some cases, but are there any calls of this method without any argument in the PR code?

Reply:
Yes. In fact, that's the reason why the `acl annotate_transaction` output is currently broken.
In fact, as far as I can tell this code is only really used in cachemgr, and is mildly overengineered.

Reply:
> Alex: does PR code ever call this method without arguments (i.e. relying on the CRLF default)? ... are there any calls of this method without any argument in the PR code?

> Francesco: Yes.

Where?

Reply:
Doh, sorry, I got confused.
There used to be one such caller, ACLAnnotationData::dump. The PR removes it, so now AFAICT there's none.